### PR TITLE
add links to GH & intro materials (closes #85 & #82)

### LINKED ISSUE #85: Link to OSF intro material
Link to https://cos.io/our-products/open-science-framework/ in addition to osf.io? Might give a better idea to total newcomers what this is all about?

### PR DESCRIPTION
**Pull request by gedankenstuecke**
Added the GH link to the readthedocs front page and the intro materials of OSF to the README + readthedocs.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -16,9 +16,11 @@ As such the OSF hosts large data sets, associated with papers or
 scientific projects, that can be freely downloaded. The *osfclient*
 allows people to store and retrieve large datasets associated to their
 scientific projects and papers on the OSF via the command line
-interface.
+interface. If you are completely new to the OSF you can
+read the `OSF introductory materials`.
 
-This is a very new project, it has some rough edges.
+This is a very new project, it has some rough edges. You can find the code for
+the osf-cli in the `GitHub repository`.
```
Comment (betatim) [inline docs/index.rst]: can you make this ``osfclient`` instead?

**Comment by betatim** [comment]
Good except for that one nitpick. Merge after.

**Comment by gedankenstuecke** [comment]
There you go!

**Comment by betatim** [comment]
Super :star: !

**Comment by gedankenstuecke** [comment]
Always happy to make a small contribution! 🎉

**Pull request merged**